src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -73,26 +73,31 @@ export type SourceInformation = LocalCacheSourceInformation | GithubSourceInform
 
 interface BaseSourceInformation {
 	type: string;
+	referenceId: string; // Dictates how a supporting tool will locate and download a given feature. See https://github.com/devcontainers/spec/blob/main/proposals/devcontainer-features.md#referencing-a-feature
```

Comment:
In other places in the codebase we've used `userFeature` (which has an `id` and `options`) to represent the entire object in the `devcontainer.json`'s `features` map.

I'm thinking here we want to try and align the names as much as possible, either by changing what we already have, or choosing a name here closer to other, existing uses

Reply:
Renamed to `userFeatureId`